src/im_tables/views.cljs

```diff
@@ -46,7 +46,21 @@
                                             (:class vocab) "/"
                                             (:objectId vocab)))}}})
 
-
+(def biotestmine-config
+  {:service {:root "http://localhost:8080/biotestmine"}
+   :query {:from "Gene"
+           :select ["symbol"
+                    "secondaryIdentifier"
+                    "dataSets.description"
+                    "primaryIdentifier"
+                    "organism.name"
+                    "dataSets.name"]}
+   :settings {:pagination {:limit 10}
+              :links {:vocab {:mine "BananaMine"}
```

Comment:
Why is this banana? My guess is copy/paste since I know we have Banana elsewhere in the code - but maybe biotestmine would make more sense?

Reply:
Yeah, it's bananamine for all the other mine configs, and I just copied one of them and changed the root address.
Perhaps, but it doesn't really matter since the links are all just illustrative.

Reply:
let's fix it anyway to keep things tidy.

Reply:
Ok